### ISSUE TITLE
BUG: cannot connect to WebDAV server on non-standard port

### ISSUE THREAD

**Issue by cairijun** [kolyvan/kybook#252]
I have a WebDAV server deployed on a non-standard port: `https://my.webdav.server:4433/webdav/`, but bybook3 seems to connect to the standard port `443`, which results in `NSURLErrorDomain` with error code `-1004`.

**Comment by s7ev3n** [comment]
Me too, I met the same error.

**Comment by jtgtifh** [comment]
Yes,you are right.Kybook only supports standard webdav port 443(https). If you build your own webdav, you need to change the port to 443. In addition, to access with a domain name, you need to apply for a https certificate for your domain name (e.g. Let's Encrypt); to access with a local address, you need to make your own https certificate using a software called mkcert (there are tutorials on the web).
